prov/psm/src/psmx_init.c

```diff
@@ -398,7 +399,7 @@ PSM_INI
 	int check_version;
 	int err;
 
-	fi_log_init();
+	psm_handle = fi_log_register_provider("psm");
```

Comment:
Supporting dynamic providers is going to be important for Cisco at some point in the future, so I'd really like to retain logging functionality in that case.